src/shared.js

```diff
@@ -0,0 +1,78 @@
+import dotenv from "dotenv";
+import {REST} from "@discordjs/rest";
+import {Constants, DefaultRestOptions} from "./constants.js";
+import yargs from "yargs";
+import {readdirSync} from "fs";
+
+let envConfigured = false;
+const commandsFolder = Constants.commandsFolder;
+const jsExt = Constants.jsExt;
+
+function validateEnvConfigState() {
+    if (!envConfigured) throw new Error("Please configure the environment first using 'shared:configureEnv()'. ");
+
+    return true;
+}
+
+export function registerRejectionHandler() {
+    process.on('unhandledRejection', (err) => {
+        console.log('Unknown error occurred:\n');
+        console.log(err);
+    });
+}
+
+export function configure() {
+    dotenv.config();
+    registerRejectionHandler();
+
+    envConfigured = true;
+}
+
+export function processArgs() {
+    return yargs(process.argv).argv;
+}
+
+export function getEnvConfig() {
+    validateEnvConfigState();
+
+    let {CLIENT_ID, GUILD_ID, TOKEN, MONGODB} = process.env;
+
+    if (!MONGODB) {
+        MONGODB = "mongodb://127.0.0.1:27017/nickdb";
```

Comment:
```suggestion
        MONGODB = "mongodb://127.0.0.1:27017/NickNelson";
```

Reply:
Also should this not be using the default database name anyway?

Reply:
I saw it! Thanks.